pkg/lintcontext/parse_yaml.go

```diff
@@ -80,10 +82,9 @@ func (l *lintContextImpl) renderHelmChart(dir string) (map[string]string, error)
 	if err := chrt.Validate(); err != nil {
 		return nil, err
 	}
-	valOpts := &values.Options{ValueFiles: []string{filepath.Join(dir, "values.yaml")}}
```

Comment:
This code isn't necessary since the Helm parser already uses the default `values.yaml` within the chart when it executes the render.